Android build error

Related to https://github.com/minetest/minetest/commit/b662a4577d692329b9ca83525e6039f2ddcd1ac1

```jni/../jni/src/touchscreengui.cpp: In member function 'bool TouchScreenGUI::doubleTapDetection()':
jni/../jni/src/touchscreengui.cpp:925:71: error: 'getTimeMs' was not declared in this scope
  u32 delta = porting::getDeltaMs(m_key_events[0].down_time, getTimeMs());
                                                                       ^
jni/../jni/src/touchscreengui.cpp:925:71: note: suggested alternative:
In file included from jni/../jni/src/touchscreengui.cpp:27:0:
jni/../jni/src/porting.h:236:12: note:   'porting::getTimeMs'
 inline u64 getTimeMs()
            ^
jni/../jni/src/touchscreengui.cpp: In member function 'void TouchScreenGUI::step(float)':
jni/../jni/src/touchscreengui.cpp:1009:61: error: 'getTimeMs' was not declared in this scope
   u32 delta = porting::getDeltaMs(m_move_downtime,getTimeMs());
                                                             ^
jni/../jni/src/touchscreengui.cpp:1009:61: note: suggested alternative:
In file included from jni/../jni/src/touchscreengui.cpp:27:0:
jni/../jni/src/porting.h:236:12: note:   'porting::getTimeMs'
 inline u64 getTimeMs()
            ^
/home/home/android-ndk-r11c/build/core/build-binary.mk:462: recipe for target 'obj/local/armeabi-v7a/objs/minetest/jni/src/touchscreengui.o' failed
make[2]: *** [obj/local/armeabi-v7a/objs/minetest/jni/src/touchscreengui.o] Error 1

--- a/src/touchscreengui.cpp
+++ b/src/touchscreengui.cpp
@@ -922,7 +922,7 @@ bool TouchScreenGUI::doubleTapDetection()
 	m_key_events[1].x         = m_move_downlocation.X;
 	m_key_events[1].y         = m_move_downlocation.Y;
 
-	u32 delta = porting::getDeltaMs(m_key_events[0].down_time, getTimeMs());
+	u32 delta = porting::getDeltaMs(m_key_events[0].down_time, porting::getTimeMs());
 	if (delta > 400)
 		return false;
 
@@ -1006,7 +1006,7 @@ void TouchScreenGUI::step(float dtime)
 			(!m_move_has_really_moved) &&
 			(!m_move_sent_as_mouse_event)) {
 
-		u32 delta = porting::getDeltaMs(m_move_downtime,getTimeMs());
+		u32 delta = porting::getDeltaMs(m_move_downtime,porting::getTimeMs());
 
 		if (delta > MIN_DIG_TIME_MS) {
 			m_shootline = m_device

--- a/src/touchscreengui.cpp
+++ b/src/touchscreengui.cpp
@@ -1006,7 +1006,7 @@ void TouchScreenGUI::step(float dtime)
 			(!m_move_has_really_moved) &&
 			(!m_move_sent_as_mouse_event)) {
 
-		u32 delta = porting::getDeltaMs(m_move_downtime,porting::getTimeMs());
+		u32 delta = porting::getDeltaMs(m_move_downtime, porting::getTimeMs());
 
 		if (delta > MIN_DIG_TIME_MS) {
 			m_shootline = m_device

